ateam_ui/src/src/state.ts

```diff
@@ -57,6 +58,55 @@ export class AppState {
     selected_play_name: string = null;
     override_play_in_progress: string = null;
 
+    getSTPParams() {
+        const state = this; // fix dumb javascript things
+
+        this.ros.getParams(function(param_list) {
+            for (let name of param_list) {
+                // Check if the parameter is an stp param
+                if (name.includes("/kenobi_node:stp_parameters")) {
+
+                    const substrings = name.split(".");
+
+                    // /kenobi_node:stp_parameters.TestKickPlay.use_pivot_kick
+                    // Make sure param has play name and stp param name
+                    if (substrings.length < 3) {
+                        continue;
+                    }
+
+                    // Expected format is: "/kenobi_node:stp_parameters.PlayName.ParamName"
+                    const play_name = substrings[1];
+                    const stp_param_name = substrings[2];
```

Comment:
This appears to assume all parameters are at the play level. Parameters can be created anywhere in the STP tree, so in reality we might have something like this: `stp_parameters.StopPlay.EasyMoveTo.MaxVelocity`.
How hard would it be to show that full tree?